docs/install.md

```diff
@@ -332,6 +332,11 @@ for a complete list of supported properties.
 
 Here is a minimal example driver deployment created with a custom resource:
 
+**NOTE**: Change `deviceMode` to `direct` if PMEM-CSI was installed using
```

Comment:
The PMEM-CSI driver was not installed yet. Creating the CR will install it. The PMEM-CSI operator was installed, but it doesn't have a device mode.

Sorry, I'm confused about what this is meant to tell me :sweat_smile: